job-dsls/jobs/kie_jenkinsScripts_PR.groovy

```diff
@@ -21,7 +21,7 @@ cd job-dsls
 def errorSh='''
 touch trace.sh
```

Comment:
Should the directory change to `$WORKSPACE` be here as well?
```suggestion
cd $WORKSPACE
touch trace.sh
```

Reply:
that could be the missing piece :)

Reply:
you're absolutely right